lang/cs/Org.Apache.REEF.Wake/Remote/Proto/WakeRemoteProtosGen.cs

```diff
@@ -29,27 +29,27 @@
 // Generated from: src/main/proto/RemoteProtocol.proto
 namespace Org.Apache.REEF.Wake.Remote.Proto
 {
-  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name=@"WakeMessagePBuf")]
+  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name = @"WakeMessagePBuf")]
```

Comment:
Does it make sense to update autogenerated files (both proto and avro)? I understand that we don't generate them on-the-fly like we do for Java, but once we re-generate them, we'll have to update them again... Is there a way to exclude them from StyleCop check?